csharp/src/Google.Protobuf.Conformance/BUILD.bazel

```diff
@@ -0,0 +1,97 @@
+load("//build_defs:internal_shell.bzl", "inline_sh_binary")
+load("@rules_pkg//:mappings.bzl", "pkg_files", "strip_prefix")
+
+inline_sh_binary(
+    name = "build_conformance_test",
+    srcs = ["Google.Protobuf.Conformance.csproj"],
+    cmd = "dotnet build -c Release $(SRCS)",
+    visibility = ["//visibility:private"],
+)
+
+CONFORMANCE_DEPS = [
```

Comment:
I know nothing about Bazel, but it does seem odd that no package versions are specified here.

Reply:
Yea the current state is a bit weird.  What I've done (for now) is basically just turned Bazel into a wrapper around `dotnet`.  So it'll use whatever versions are specified by the project file.